package.json

```diff
@@ -22,23 +22,14 @@
   },
   "dependencies": {
     "moment-timezone": "^0.6.0",
-    "rrule": "2.8.1",
-    "uuid": "^11.1.0"
+    "rrule": "^2.8.1"
```

Comment:
Please keep rrule to "2.8.1" since we had several issues with rrule in the past and have to review each and every single update very closely!

Reply:
Oh, okay. I just undone this.